src/modules/modCommands/processCommands.js

```diff
@@ -50,14 +50,9 @@ async function processCommandsNewMessage(client, message) {
         await cmd.fun(client, message, args, level);
     } catch (e) {
         console.error(`Error running command "${command}":\n${e?.stack || e}`);
-        if (message.deleted)
-            message.channel
-                .send(`Something went wrong processing that command! Please try again later.`)
-                .catch((err) => console.error('Error sending error!\n', err));
-        else
-            message
-                .reply(`Something went wrong processing that command! Please try again later.`)
-                .catch((err) => console.error('Error sending error!\n', err));
+        message.channel
+            .send(`Something went wrong processing that command! Please try again later.`)
+            .catch((err) => console.error('Error sending error!\n', err));
```

Comment:
Unnecessary but ok.

Reply:
the one before was unnessesary code dublication

Reply:
Well it was replying or sending

Reply:
does it really matter if it sends or replies? and this is a lot smaller

Reply:
I guess not